release/maintain/index.html

```diff
@@ -1,8 +1,8 @@
 ---
-layout: release_28
+layout: release_29
 title: GeoServer
-version: 2.8.5
-jira_version: 13200
-release_date: August 12, 2016
-announce: 2016/08/12/geoserver-2-8-5-released/
+version: 2.9.2
+jira_version: 13001
+release_date: October 12th, 2016
+announce: 2016/10/12/geoserver-2-9-2-released/
```

Comment:
This is tricky as 2.9.2 was a stable release - sensible to mark it as the maintenance series now.